src/assets/js/index.js

```diff
@@ -0,0 +1,131 @@
+/* eslint-disable import/no-cycle */
```

Comment:
it may be useful to comment on your code often especially the function to explain a little further the return or the purpose of the function. It may help with readability